languages/swift/exercises/concept/structs-and-classes/.docs/introduction.md

```diff
@@ -0,0 +1,158 @@
+Structs and classes are two of the primary building blocks of Swift programming. They are both means of grouping together related data and functions into self-contained units of functionality. All of the basic types that you have been working with up to this point, such as `Int`, `Double`, `Bool`, and `Character` as well as collection types like `Array`, `String`, and `Dictionary` are actually structs. And when you define a struct or class, you are defining a new type to be used within Swift, just as you used those types.
+
+There are many similarities between structs and classes in Swift. Among other similarities, both are able to store values in _properties_ and provide functionality through the use of _methods_.
+
+They each provide some additional functionality, which is out of scope for this exercise.
+
+### Defining structs and classes
+
+Both structs and classes are defined in roughly the same way. They start with the appropriate keyword followed by the type name that is being defined, then the body of the struct/class follows, placed between curly braces.
+
+The body may consist of stored properties, which are defined and behave just like regular constants or variables.
+
+```swift
+struct CharacterStats {
+  var health = 0.0
+  var speed = 0
+  var strength = 0
+}
+
+class GameCharacter {
+  var stats = CharacterStats()
+  var characterClass: String?
+  var name: String?
+  var active = false
+  let id = makeRandomID()
+}
+```
+
+Here we have a struct that contains the statistics that will be associated with a character in a game and a struct that contains information about the character. Note that the `Character` class contains an instance of the `CharacterStats` struct inside of it.
```

Comment:
> Here we have a struct that contains the statistics that will be associated with a character in a game and a struct that contains information about the character.

I feel like this can maybe be omitted.

Reply:
Removed